lib/render/layout.php

```diff
@@ -62,11 +55,23 @@ function RenderSharedHeader()
     echo "<script src='https://ajax.googleapis.com/ajax/libs/jquery/1.8.1/jquery.js'></script>\n";
     echo "<script src='https://ajax.googleapis.com/ajax/libs/jqueryui/1.8.23/jquery-ui.js'></script>\n";
 
+    // jQuery Modal
+    echo "<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery-modal/0.9.1/jquery.modal.min.js'></script>";
+    echo "<link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/jquery-modal/0.9.1/jquery.modal.min.css' />";
+
+    echo "<script src='https://cdnjs.cloudflare.com/ajax/libs/knockout/3.5.0/knockout-min.js'></script>";
+
     //    jQuery, and custom js
     //echo "<script src='//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js'></script>\n";
     //echo "<script src='/vendor/jquery-ui-1.10.2.custom.min.js'></script>\n";
     echo "<script src='/js/all.js?v=" . VERSION . "'></script>\n";
     echo "<script>window.assetUrl='" . getenv('ASSET_URL') . "'</script>\n";
+
+    echo "<link rel='stylesheet' href='/css/styles.css?v=" . VERSION . "' media='screen'>\n";
```

Comment:
I there a technical reason for moving the styles to the bottom of the list? web vitals?

Reply:
Yep, sure is. We don't want vendor CSS to override site CSS in such a way that requires us to use `!important`, which I consider bad code smell if used willy nilly. The best way to prevent this is to include our site CSS after all vendor CSS.

And in this case, the practical reason is because if I don't do this, the menu bar at the top of the site overlaps the jQuery modals and makes them look *really* funky. The only way to fix it without reorganizing the CSS was to use `!important` on the menu's z-index, but that felt really hacky to me.